### DISCUSSION TITLE
Transition Rate and Hyperfine

### DISCUSSION THREAD

**Discussion by Mrins576** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#133] [Q&A] [answered]
How can I get Transition Rate which include the hyperfine structure since I could not find such module in ARC

**Comment by nikolasibalic** [comment, marked as answer]
Hi @Mrins576 , thank you for your question.
You can get what you want by combining following information:
- fine basis transition rates (see [`getTransitionRate`](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/generated/arc.alkali_atom_functions.AlkaliAtom.getTransitionRate.html)); and
- hyperfine state branching ratios (see [`getBranchingRatio`](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/generated/arc.alkali_atom_functions.AlkaliAtom.getBranchingRatio.html)).

**Reply by Mrins576** [reply]
Can you suggest how to find the transition rate between 5P1/2, F=3 and 5S1/2, F=2.
